src/DynamoCoreWpf/Views/HomePage/HomePage.xaml.cs

```diff
@@ -520,7 +520,7 @@ internal void OpenWorkspace()
                 return;
             }
 
-            this.startPage?.DynamoViewModel?.ShowOpenDialogAndOpenResultCommand.Execute(null);
+            this.startPage?.DynamoViewModel?.ShowOpenDialogAndOpenResultAsync(null);
```

Comment:
@RobertGlobant20 I think there is difference between command and your new function, for example analytics tracking. Would you double check?

Reply:
I've fixed the functionality for using Commands, so I had to add two more commands, one for Templates and the other one for Opening a file.

Reply:
I was checking that the Analytics logging is done after the command is executed then I think there should not be any problem.
![image](https://github.com/DynamoDS/Dynamo/assets/61755417/9bedde34-2c04-4e30-a4bc-d54a7364fba0)
![image](https://github.com/DynamoDS/Dynamo/assets/61755417/f2e3f62b-d7b3-4a6e-950b-03782f391d4b)